toolchain/parser/parser.h

```diff
@@ -262,6 +262,17 @@ class Parser {
   // parses should only need to look down a couple steps.
   auto GetDeclarationContext() -> DeclarationContext;
 
+  // Handles of recovery in a declaration, particularly before any possible
```

Comment:
Is there a missing or extra word near the start of this sentence?

Reply:
of -> error